src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -236,6 +237,7 @@ object ErrorAggregator {
       dimensions("quantum_ready") = meta.isQuantumReady
       dimensions("experiment_id") = experiment_id
       dimensions("experiment_branch") = experiment_branch
+      dimensions("profile_age_days") = meta.`environment.profile`.flatMap(_.ageDaysBin(new DateTime()))
```

Comment:
It makes me a bit nervous to tie profile age to the current server clock, in case we ever replay/backfill data -- is there a specific reason we want to use this instead of submission date?

Reply:
Good catch!